WcaOnRails/app/models/locale.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+class Locale < SimpleDelegator
+  include ActionView::Helpers::TextHelper
+  SOME_CHARS = '[\s\S]*?'
```

Comment:
Yeah, it means really any possible character. IIRC it should work with a dot and the `m` flag, but I don't think it would be much better. Also, note that this one was made in JavaScript, where the above is the best solution.